expertino/clips/refbox-recv.clp

```diff
@@ -27,11 +27,11 @@
   (bind ?new-team-other ?team-other)
   (bind ?new-points ?points)
   (bind ?new-points-other ?points-other)
-  (retract ?gt)
+;  (retract ?gt)
   (bind ?time (pb-field-value ?p "game_time"))
   (bind ?sec (pb-field-value ?time "sec"))
   (bind ?nsec (pb-field-value ?time "nsec"))
-  (assert (game-time (+ ?sec (/  ?nsec 1000000))))
+;  (assert (game-time (+ ?sec (/  ?nsec 1000000))))
```

Comment:
why do we get rid of the notation of game-time?
Is it not a useful piece of information?